### PR TITLE
Documentation additions and updates

### PR DESCRIPTION
**Pull request by adam-palazzo**
I went ahead and added the code of conduct and contributing files. I also made some small edits to the readme and json @RichardLitt

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,28 @@
...
 [![npm version](https://badge.fury.io/js/orbit-db-cli.svg)](https://www.npmjs.com/package/orbit-db-cli)
 [![node](https://img.shields.io/node/v/orbit-db-cli.svg)](https://www.npmjs.com/package/orbit-db-cli)
 
+> CLI for orbit-db
+
 A CLI tool to manage [orbit-db](https://github.com/orbitdb/orbit-db) databases.
 
 <a href="https://asciinema.org/a/JdTmmdBCZarkBkPqbueicwMrG" target="_blank"><img src="https://asciinema.org/a/JdTmmdBCZarkBkPqbueicwMrG.png" width="50%"/></a>
 
+- [Quick Start](#quick-start)
```
Comment (RichardLitt) [inline README.md]: Can we add a `## Table of Contents` above this?

Comment (adam-palazzo) [reply README.md]: We can!

**Comment by adam-palazzo** [comment]
@RichardLitt I made the requested changes

**Comment by RichardLitt** [comment]
Thanks.

**Pull request merged**